dffml/operation/source.py

```diff
@@ -0,0 +1,92 @@
+from typing import List, Dict, Any
+
+from ..df.base import op
+from ..base import config
+from ..df.types import Definition
+from ..source.source import BaseSource
+
+
+@config
+class SourceOperationConfig:
+    source: BaseSource
+
+
+@op(
+    inputs={
+        "features": Definition(name="features", primitive="List[str]"),
+        "predict_features": Definition(
```

Comment:
We don't seem to use `predict_features` anywhere, what's the point in having it?

Reply:
We are actually using this when we do not need to process some of the features in the dataset, like the prediction value or the classification value.